src/components/JoinUs/elements.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import ReactMarkdown from 'react-markdown'
+import styled from 'styled-components'
+import { Padding } from 'styled-components-spacing'
+import breakpoint from 'styled-components-breakpoint'
+import { Grid, Row, Col } from '../grid'
+import { H3, Paragraph } from '../Typography'
+import GreyBG from '../GreyBG'
+
+/**
+ * Layout
+ */
+export const FirstColumn = ({ children }) => (
+  <Col width={[1, 1, 1, 1, 6 / 12, 5 / 12]}>{children}</Col>
+)
+
+export const SecondColum = ({ fillWidth = false, children }) => (
+  <ShiftedColumn
+    width={fillWidth ? [1, 1, 1, 1, 6 / 12] : [1, 1, 1, 1, 6 / 12, 5 / 12]}
+  >
+    {children}
+  </ShiftedColumn>
+)
+
+export const List = ({ list }) => (
+  <Padding top={1}>
+    <ReactMarkdown
+      renderers={{
+        heading: props => <Paragraph noMargin bold {...props} />, // eslint-disable-line react/display-name
```

Comment:
I reckon this should use a heading component (not sure which size), rather than `Paragraph`?

Reply:
Maybe you're right

Reply:
In terms of style, it looks like an `h5` - small title, I'll change to that

Reply:
You're totally right, we shouldn't skip from h2 to h5. It should be `as="hX"` where X is the next number down in the semantic order (for accessibility reasons)

I think it would be helpful if we called our typographic components after the designers' names, rather than being tied to the semantic names.

Reply:
ok!

Reply:
done